packages/ketcher-react/src/script/ui/views/components/StructEditor/StructEditor.jsx

```diff
@@ -212,6 +215,17 @@ class StructEditor extends Component {
           <div
             ref={this.editorRef}
             className={clsx(classes.intermediateCanvas)}
+            onScroll={throttle(40, () => {
+              if (!this.editor.rotateController.handle) {
+                return
+              }
+              const handleCenter = this.editor.rotateController.handleCenter
+              const handleCenterInViewport =
+                this.editor.render.raphael2View(handleCenter)
+              this.editor.event.updateFloatingTools.dispatch(
+                updateFloatingToolsPos(handleCenterInViewport)
+              )
```

Comment:
Could I propose moving the implementation of handling the scrolling inside the Editor.ts to an additional public method, so this logic will be encapsulated close to rotateController and rendering?